edXRegressionAutomated/Tests/LMS/pages/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+AUTH_USER =os.environ['AUTH_USER_NAME']
```

Comment:
If a user has not set the username or password, he or she should find out quickly. You should check that these are set, and error out if not.

(It's actually better if you set it to a default. But considering that defaults won't work for these kinds of settings, you should produce an error.)